up.go

```diff
@@ -75,13 +75,32 @@ var (
 	unsafeMode   = pflag.Bool("unsafe-full-throttle", false, "enable mode in which command is executed immediately after any change")
 	outputScript = pflag.StringP("output-script", "o", "", "save the command to specified `file` if Ctrl-X is pressed (default: up<N>.sh)")
 	debugMode    = pflag.Bool("debug", false, "debug mode")
+	versionFlag = pflag.Bool("version", false, "display version and credits")
 )
 
 func main() {
 	// Handle command-line flags
 	pflag.Parse()
 
 	log.SetOutput(ioutil.Discard)
+	if *versionFlag {
+		os.Stdout.WriteString("up: Ultimate Plumber v" + version + " https://github.com/akavel/up\n")
```

Comment:
Please add your email and name (or other preferred identifier) in the AUTHORS file! :) :heart: